lighthouse-cli/test/smokehouse/perf/expectations.js

```diff
@@ -47,6 +47,9 @@ module.exports = [
       },
       'uses-rel-preconnect': {
         score: '<1',
+        warnings: {
```

Comment:
regex on the origin in the warning string? Or too annoying

Reply:
good call done